.github/integration/setup/common/10_services.sh

```diff
@@ -14,19 +14,21 @@ if [ "$STORAGETYPE" = s3 ]; then
     tostart="mq db s3 certfixer"
 fi
 
+# we need to let it be split here otherwise it takes it as one
```

Comment:
This comment was a bit hard to read!
maybe something like
```suggestion
# We need to leave the $tostart variable unquoted here since we want it to split
```